src/components/MaterialUI/Inputs/SearchControl.js

```diff
@@ -0,0 +1,175 @@
+import React, { useState, useRef } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import SelectProps from "./SelectProps";
+import Icon from "./../DataDisplay/Icon";
+import IconButton from "@material-ui/core/IconButton";
+import Select from "./Select";
+import Input from "@material-ui/core/Input";
+
+export const useStyles = makeStyles(theme => ({
+	clearButton: {
+		fontSize: theme.spacing(2.2),
+		padding: theme.spacing(0.3),
+		marginLeft: "0 !important",
```

Comment:
Will it work without important? It should

Reply:
The style from "MuiInputBase-root" takes precedence if I don't put important. Any suggestions ? I would rather not changing it in muiThems.js of course.

Reply:
Hm, ok.. that's strange coz we have many places were we are overriding global classes by this way without important. but, if it doesn't work without it left it as it is. I don't think it make sense to spend time on it.